ext/pcre/php_pcre.stub.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/** @return int|false */
+function preg_match(string $pattern, string $subject, &$subpatterns = null, int $flags = 0, int $offset = 0) {}
+
+/** @return int|false */
+function preg_match_all(string $pattern, string $subject, &$subpatterns = null, int $flags = 0, int $offset = 0) {}
+
+/**
+ * @param string|array $regex
+ * @param string|array $replace
+ * @param string|array $subject
+ * @return string|array|null
```

Comment:
`false` is also a possible return value.

Reply:
Also for preg_filter().

Reply:
I've eliminated this return type in https://github.com/php/php-src/commit/228bae7d74a7886f05716ef5ac0f3ba200caccc4.